morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradeConfigAndContext.java

```diff
@@ -0,0 +1,45 @@
+package org.alfasoftware.morf.upgrade;
+
+import java.util.Set;
+
+import com.google.common.base.Preconditions;
+
+/**
+ * Configuration and context bean for the {@link Upgrade} process.
+ *
+ * @author Copyright (c) Alfa Financial Software Limited. 2024
+ */
+public class UpgradeConfigAndContext {
+
+  /**
+   * Set of full names of upgrade step classes which should be executed non-parallel.
+   * No other other upgrade step will be running while listed upgrade steps are being executed.
+   * This impacts Graph-Based Upgrade only.
+   */
+  private Set<String> exclusiveExecutionSteps = Set.of();
+
+  /**
+   * A schema change adaptor to be used during upgrade.
+   */
+  private SchemaChangeAdaptor upgradeChangeAdaptor = new SchemaChangeAdaptor.NoOp();
```

Comment:
pretty minor, but the variable being named upgradeChangeAdaptor when the type is schemaChangeAdaptor could perhaps add a slightly mild hint of momentary confusion that could be avoided

Reply:
(same in public method on this class)

Reply:
Well spotted.